packages/main/package.json

```diff
@@ -35,6 +35,7 @@
     "concurrently": "^6.2.1",
     "cross-env": "^7.0.3",
     "css-loader": "^6.2.0",
+    "dotenv-webpack": "^8.0.1",
```

Comment:
why did you install this?
are envs not working already?

Reply:
Yes boss. Envs are not loaded. I had to configure that plugin

Reply:
but envs are already in use in the app @billmal071

Reply:
@ickynavigator Our intial PR was merged and even the live server couldn't load the env variables boss

Reply:
Yes, they are. They need to give the env variables to Mayowa.

Reply:
@ibitolamayowa

Reply:
@billmal071 how does it load env vars and where do I store it locally to test because when I did, it kept failing everytime

Reply:
We'll update you.